30_data_summarize/src/site_summary.R

```diff
@@ -0,0 +1,33 @@
+#one row per site, including ID, lat/lon, site type, maybe n records from the data pull
+
+summarize_site_observations <- function(target_ind, daily_flow_ind, 
+                                        dv_inventory, uv_inventory) {
+  
+  daily_flow <- readRDS(sc_retrieve(daily_flow_ind, remake_file = '20_data_munge.yml')) 
+  #daily_flow <- readRDS('20_data_munge/out/daily_flow.rds')
+  #dv_inventory <- readRDS('10_nwis_pull/inout/nwis_dv_inventory.rds')
+  #uv_inventory <- readRDS('10_nwis_pull/inout/nwis_uv_inventory.rds')
+  #dv_inventory <- readRDS(sc_retrieve(dv_inventory_ind, remake_file = '20_data_munge.yml'))
+  #uv_inventory <- readRDS(sc_retrieve(uv_inventory_ind, remake_file = '20_data_munge.yml'))
```

Comment:
This seems like content from when you were testing this that you should delete?

Reply:
done